dts/bindings/gpio/altr,pio-1.0.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Altera PIO node
+
+compatible: "altr,pio-1.0"
+
+include: [gpio-controller.yaml, base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  direction:
+    type: string
+    description: |
+      Direction can be set to Bidir, Input, InOut or Output.
+    default: output
+    enum:
+      - "bidir"
+      - "input"
+      - "inout"
+      - "output"
```

Comment:
Please elaborate on the consequences of setting this property in the description.